Tasks/UseDotNetV2/package.json

```diff
@@ -29,7 +29,7 @@
     "azure-pipelines-task-lib": "3.0.0-preview",
     "azure-pipelines-tool-lib": "0.11.0",
     "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
-    "semver": "6.0.0",
+    "semver": "6.3.0",
     "typed-rest-client": "0.11.0",
```

Comment:
You need to put caret here

Reply:
I had removed the carat in the master PR as well, so it got cherry-picked here.